drivers/ps2/Kconfig.xec

```diff
@@ -0,0 +1,27 @@
+# Kconfig.xec - Microchip XEC PS2 configuration options
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig PS2_XEC
+	bool "XEC Microchip PS2 driver"
+	help
+	  Enable the Microchip XEC PS2 driver.
+
+config PS2_XEC
```

Comment:
As this duplicates the same `PS2_XEC` symbol as the menuconfig above, would this work? I think kconfig would complain about this.